src/account-info/LogoutButton.js

```diff
@@ -30,6 +31,7 @@ class LogoutButton extends PureComponent {
       try {
         await unregisterPush(auth, pushToken);
       } catch (e) {
+        logErrorToSentry(e);
         console.log('failed to unregister Push token', e); // eslint-disable-line
```

Comment:
Why would you want to keep both `logError` and `console.log`? 
And why would you want them to be different?

Instead, make `logError` do a Sentry send or console.log depending on `enableSentry`.